hack/ci/e2e.sh

```diff
@@ -106,9 +106,16 @@ apiVersion: kind.sigs.k8s.io/v1alpha3
 nodes:
 # the control plane node
 - role: control-plane
+  extraMounts:
+  - containerPath: /etc/kubernetes/audit-policy.yaml
+  - hostPath: /tmp/audit-policy.yaml
```

Comment:
@neolit123 we now generate/mount the policy using /tmp

Reply:
i think now the policy has to be passed to the API server along side other flags?

```
kind: Cluster
apiVersion: kind.sigs.k8s.io/v1alpha3
kubeadmConfigPatches:
- |
  apiVersion: kubeadm.k8s.io/v1beta1
  kind: ClusterConfiguration
  metadata:
    name: config
  apiServer:
    extraArgs:
      "audit-policy-file": "/etc/kubernetes/audit-policy.yaml"
nodes:
...
```

Reply:
Yep, added.